pygradle-plugin/build.gradle

```diff
@@ -26,6 +40,17 @@ sourceSets {
         compileClasspath += sourceSets.main.output + configurations.testRuntime
         runtimeClasspath += output + compileClasspath
     }
+
```

Comment:
That was my attempt of getting them to run under 4 gigs of ram.  Since it didn't work, i can merge them back together.